lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -656,7 +656,7 @@ public function get_item_schema() {
 				'slug'        => array(
 					'description' => 'An alphanumeric identifier for the object unique to its type.',
 					'type'        => 'string',
-					'context'     => array( 'embed', 'view', 'edit' ),
+					'context'     => array( 'view', 'edit' ),
```

Comment:
> Is there a reason besides consistency?

Not at the top of my head, but I'm sleep deprived.

We can just see if it comes up again.